### ISSUE TITLE
Please help to use it, i'm a beginner :(

### ISSUE THREAD

**Issue by pawpsicle** [PokemonGoF/PokemonGo-Bot#2773]
i'm a beginner and i dont understand how to use it, i read wiki but i cant understand it, i very grateful for some one can explain it clearly or have video tutorial to use it, it just not for me, but it for me and many people cant understand how to use it

**Comment by Bramzor** [comment]
Did you even check youtube?

**Comment by pawpsicle** [comment]
no, i search it but cant found it, you have tutorial in youtube ? can give link ? please

**Comment by EricFelbs** [comment]
https://youtu.be/KiL579gt310
